File: Data Structures & Algorithms/two-integer-sum/submission-3.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
            int[] n2 = new int[2];
        for(int i=0;i<nums.length;i++){
            int n = nums[i];
            for(int k=0;k<nums.length;k++){
            for(int j=0;j<nums.length;j++){
                if(i == j){
                    continue;
                }
                int n1=n+nums[j];
                if(n1==target){
                   // int[] n2 = new int[2];
                    n2[0]=j;
                    n2[1]=i;
                   // return n2;

                }
               

            }
            }
        }
        return n2;
    }

}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-3.py
from typing import List
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        s1={}
        final=[]
        for i in nums:
            s1[i]=s1.get(i,0)+1
        for i in range(k):
            max_key=max(s1,key=s1.get)
            final.append(max_key)
            del s1[max_key]

            if len(final)==k:
                return final
        # else:
        #     return nums[:k]
        
            

            
        